Source/WebKit/UIProcess/WebBackForwardCache.h

```diff
@@ -48,7 +48,7 @@ class WebBackForwardCache final : public CanMakeCheckedPtr<WebBackForwardCache>
     explicit WebBackForwardCache(WebProcessPool&);
     ~WebBackForwardCache();
 
-    void setCapacity(unsigned);
+    void setCapacity(WebProcessPool&, unsigned);
```

Comment:
I don't think we should make this change.

Reply:
So without this change, `WebProcessPool`'s constructor will crash (via `updateBackForwardCacheCapacity()`) because we try to ref `WebProcessPool` before its obj-c object is initialized.